docs/keda-installation.md

```diff
@@ -0,0 +1,316 @@
+
+   ```bash
+   curl localhost:5001/v2/_catalog
+   ```
+You should get a result similar to this example:
+
+   ```json
+   {"repositories":["keda-manager-dev-local","unsigned/component-descriptors/kyma-project.io/module/keda"]}
+   ```
+6. Inspect the generated module template.
+
+> **NOTE:** The following sub-steps are temporary workarounds.
+
+Edit `template.yaml` and:
+- change `target` to `control-plane`
+
+   ```yaml
+   spec:
+    target: control-plane
+    ```
+> **NOTE:** This is required in the single-cluster mode only.
+
+- change the existing repository context in `spec.descriptor.component`:
+
+   ```yaml
+   repositoryContexts:      
+     - baseUrl: k3d-kyma-registry.localhost:5000/unsigned
+       componentNameMapping: urlPath
+       type: ociRegistry
+   ```
+
+7. Install the modular Kyma on the k3d cluster.
+
+> **NOTE** This installs the latest versions of `lifecycle-manager`.
+
+Use the `--template` flag to deploy the Keda module manifest from the beggining, or apply it using kubectl later.
+
+   ```bash
+   kyma alpha deploy --template=./template.yaml
+   ```
+Kyma installation is ready, but the module is not yet activated.
+
+   ```bash
+   kubectl get kymas.operator.kyma-project.io -A
+   ```
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME           STATE   AGE
+   kcp-system   default-kyma   Ready   71s
+   ```
+
+Keda module is a known module, but not yet activated.
+
+   ```bash
+   kubectl get moduletemplates.operator.kyma-project.io -A 
+   ```
+
+You should get a result similar to the following example:
+
+   ```bash
+   NAMESPACE    NAME                  AGE
+   kcp-system   moduletemplate-keda   2m24s
+   ```
+
+8.  Give Module Manager permission to install CustomResourceDefinition (CRD) cluster-wide.
+
+> **NOTE:** Module Manager must be able to apply CRDs to install modules. In the remote mode (with control-plane managing remote clusters) it gets an administrative kubeconfig, targeting the remote cluster to do so. But in the local mode (single-cluster mode), it uses Service Account and does not have permission to create CRDs by default.
+
+Run the following to make sure Module Manager's Service Account gets an administrative role:
+
+   ```bash
+   kubectl edit clusterrole module-manager-manager-role
+   ```
+
+Add the following element under `rules`:
+
+   ```yaml
+   - apiGroups:
+     - "*"
+     resources:
+     - "*"                  
+     verbs:                  
+     - "*"
+  ```
+
+> **NOTE:** This is a temporary workaround and is only required in the single-cluster mode.
+
+9. Enable Keda in the Kyma CR.
+
+   ```bash
+   kyma alpha enable module keda -c fast
+   ```
+   
+### Local mode in k3d
+
+When using a local k3d cluster, you can also use the local OCI image registry that comes with it.
+Thanks to that, you don't need to push the Keda module images to a remote registry and test the changes in the Kyma installation set up entirely on your machine.
```

Comment:
Do I understand that sentence correctly?

Thanks to that, you don't need to (...) test the changes (...) on your machine.

Or should it be:

Thanks to that, you don't need to push (...) and **_you can_** test the changes (...) on your machine.